src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const price = 40;
+
+  if (days >= 7) {
+    return price * days - 50;
+  }
 
+  if (days >= 3) {
+    return price * days - 20;
+  } else {
+    return price * days;
+  }
```

Comment:
Last else is redundant here:
```suggestion
  return price * days;
```